erpnext/selling/doctype/customer/customer.js

```diff
@@ -134,6 +134,17 @@ frappe.ui.form.on("Customer", {
 		});
 	},
 
+	sync_with_leaflink: function(frm){
+		if(frm.doc.is_leaflink_customer == 1){
+			frappe.call({
+				method: 'erpnext.erpnext_integrations.doctype.leaflink_settings.leaflink_settings.sync_customer_with_leaflink',
+				args: {
+					"customer_name": frm.doc.name
+				}
+			})
+		}
```

Comment:
Are we not showing any message after syncing the customer with leaflink ?

Reply:
Yes that gets handled in python, after the post request is made based on the status code returned by LeafLink

Reply:
- I did not see any success message when the customer is pushed to leaflink in sync_customer_with_leaflink.
- add a show_alert in the callback